Persistence/Contexts/DatabaseSeed.cs

```diff
@@ -283,7 +283,10 @@ public static void Seed(AppDbContext context, IPasswordHasher passwordHasher)
                     {
                         Email = "admin@ungspiller.no", Password = passwordHasher.HashPassword("12345678"),
                         AzureInformation = new AzureInformation
-                            {Id = 123,UserId = "test.testovitch@testnes.no", Pat = "123123asdasd", Organization = "ReleaseNotesSystem"}
+                        {
+                            Id = 123, UserId = "test.testovitch@testnes.no", Pat = "123123asdasd",
+                            Organization = "ReleaseNotesSystem"
```

Comment:
QoL change(for oss): "ReleaseNotesSystem" --> "ReleaseNoteSystem"

Reply:
Enig